conf/base.config

```diff
@@ -15,7 +15,7 @@ process {
 
   // Variable to specify the docker / singularity image tag to use
   // Picks up on use of -r <version> in nextflow command
```

Comment:
I guess we can remove this comment too

Reply:
haha, yeah I guess I should have spotted that ;)